xby2.apps.web/AngularApp/app/components/home/homeController.js

```diff
@@ -6,24 +6,60 @@ define(["angular",
     function (angular, moment, lodash, testRepo, activityRepo, projectRepo) {
 
     var homeController = function ($scope) {
-        $scope.problemData = "Star Wars\n" +
-            "Star Wars Episode V\n" +
-            "Star Wars Episode VI\n" +
-            "Star Wars Episode I\n" +
-            "Star Wars Episode II\n" +
-            "Star Wars Episode III";
+        /* needs review, may be better
```

Comment:
Add a todo comment and mention this is the object expected from the javascript repo.